docs/user/user-defined-redirects.rst

```diff
@@ -1,39 +1,116 @@
 - :ref:`user-defined-redirects:page redirects` and :ref:`user-defined-redirects:exact redirects`
   can redirect to URLs outside Read the Docs,
   just include the protocol in ``To URL``, e.g ``https://example.com``.
 
-Redirect types
---------------
 
-We offer a few different type of redirects based on what you want to do.
+Built-in redirects
+------------------
+
+This article explains the redirects that are automatically active for all Read the Docs projects and how they are useful.
+Automatic redirects on Read the Docs are supported long-term.
+
+Built-in redirects are especially useful for creating and sharing **incoming links**
+which is discussed indepth in :doc:`/automatic-redirects`.
+
+Page redirects at ``/page/``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can link to a specific page and have it redirect to your default version,
+allowing you to create links on external sources that are always up to date.
+This is done with the ``/page/`` URL prefix.
+
+For instance, you can reach the page you are reading now by going to https://docs.readthedocs.io/page/automatic-redirects.html.
+
+Another way to handle this is the ``latest`` version.
+You can set your ``latest`` version to a specific version and just always link to ``latest``.
+You can reach this page by going to https://docs.readthedocs.io/en/latest/automatic-redirects.html.
+
+
+Root URL redirect at ``/``
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A link to the root of your documentation (`<slug>.readthedocs.io/`) will redirect to the  :term:`default version`,
+as set in your project settings.
+
+This works for both readthedocs.io (|org_brand|), readthedocs-hosted.com (|com_brand|), and :doc:`custom domains </custom-domains>`.
+
+For example::
+
+    docs.readthedocs.io -> docs.readthedocs.io/en/latest/
+    www.pip-installer.org -> www.pip-installer.org/en/latest/
+
+.. warning::
+
+   This only works for the root URL, not for internal pages.
+   It's designed to redirect people from `/` to the default version of your documentation,
+   since serving up a 404 here would be bad user experience.
+
+.. note::
+   If the "develop" branch was designated as the default version,
+   then ``http://pip.readthedocs.io/`` would redirect to ``http://pip.readthedocs.io/en/develop``.
+   But, it's not a universal redirecting solution.
+   So, for example, a link to an internal page like
+   ``http://pip.readthedocs.io/usage.html`` doesn't redirect to ``http://pip.readthedocs.io/en/latest/usage.html``.
+
+   The reasoning behind this is that RTD organizes the URLs for docs so that multiple translations and multiple versions of your docs can be organized logically and consistently for all projects that RTD hosts.
+   For the way that RTD views docs,
+   ``http://pip.readthedocs.io/en/latest/`` is the root directory for your default documentation in English, not ``http://pip.readthedocs.io/``.
+   Just like ``http://pip.readthedocs.io/en/develop/`` is the root for your development documentation in English.
+
```

Comment:
```suggestion
```

Seems this both of these notes say the same thing? I don't like using examples other people control, so we should just remove it.